pkg/vault/contracts/Vault.sol

```diff
@@ -1353,45 +1356,108 @@ contract Vault is IVaultMain, VaultCommon, Proxy {
     }
 
     /**
-     * @dev Updates reserves for underlying and wrapped tokens after wrap/unwrap operation:
-     * - updates `_reservesOf`
-     * - returns the delta underlying and wrapped tokens that were deposited/withdrawn from vault reserves
+     * @notice Updates the reserves of the vault after an ERC4626 wrap (deposit/mint) operation.
+     * @dev If there are extra tokens in the vault balances, these will be added to the reserves (which, in practice,
+     * is equal to discarding such tokens). This approach avoids DoS attacks, when a frontrunner leaves vault balances
+     * and reserves out of sync before a transaction starts.
+     *
+     * @param underlyingToken Underlying of ERC4626 wrapped token
+     * @param wrappedToken ERC4626 wrapped token
+     * @param underlyingDeltaHint Amount of underlying tokens supposedly removed from the Vault
+     * @param wrappedDeltaHint Amount of wrapped tokens supposedly added to the vault
```

Comment:
```suggestion
     * @param underlyingToken Underlying token of the ERC4626 wrapped token
     * @param wrappedToken ERC4626 wrapped token
     * @param underlyingDeltaHint Amount of underlying tokens the wrapper should have removed from the Vault
     * @param wrappedDeltaHint Amount of wrapped tokens the wrapper should have added to the Vault
```
Maybe this wording is clearer? (Same below.) We are checking whether the wrapper did what it was supposed to do - matching measurement to expectations.